flask_ipblock/__init__.py

```diff
@@ -24,5 +31,7 @@ def block_before(self):
         ip = ips[0].strip()
         if ip[-1] == ',':
             ip = ip[:-1]
-        if IPNetwork.matches_ip(ip.rsplit(',', 1)[-1].strip()):
+        ip = ip.rsplit(',', 1)[-1].strip()
+
+        if IPNetwork.matches_ip(ip, use_secondary=self.use_secondary):
```

Comment:
Could also do a `with` statement here instead of introducing an option in `IPNetwork.matches_ip`. But either way is fine.